src/createCodeFileModal.ts

```diff
@@ -55,14 +55,16 @@ export class CreateCodeFileModal extends Modal {
 		submitButton.setButtonText("Create");
 		submitButton.onClick(() => this.complete());
 
-		fileNameInput.inputEl.focus();
+		// fileNameInput.inputEl.focus(); it seems unecessary if the modal is closed
 	}
 
 	async complete() {
 		this.close();
 		const parent = (this.parent instanceof TFile ? this.parent.parent : this.parent) as TFolder;
-		const newPath = `${parent.path}/${this.fileName}.${this.fileExtension}`;
-
+		let newPath = `${parent.path}/${this.fileName}.${this.fileExtension}`;
```

Comment:
I'm curious, what is the error you are experiencing?

Reply:
I am on Windows and am not getting the double slash at the beginning, so not sure what causes this, but if it solves it for you, I'm fine with merging it.